styles.css

```diff
@@ -127,7 +138,7 @@ form input {
 	font-weight: bold;
 	padding: 5px;
 	text-align: center;
-	width: 280px;
+	width: 100%;
```

Comment:
Not a major issue but can it be done to only have this set to `100%` in mobile view? The width should remain to be `280px` when displayed in the browser.